driver/InputFlags.cpp

```diff
@@ -292,6 +292,63 @@ TensorParameters InputFlags::GetValueTensor(const std::string& long_name) const
 
     MIOPEN_THROW("Too many tensor descriptor parameters.");
 }
+
+std::vector<int> InputFlags::GetValueVectorInt(const std::string& long_name) const
```

Comment:
`std::vector<int>` this can't properly support huge tensors.

Reply:
fixed it.